charts/incubator/hyperswitch-app/router.toml

```diff
@@ -0,0 +1,314 @@
+[bank_config]
```

Comment:
add comments for each config as done [here](https://github.com/juspay/hyperswitch/blob/main/config/config.example.toml)